README.md

```diff
@@ -1,247 +1,187 @@
 <h1 align="center">
-goose
+Goose is your on-machine developer agent, automating engineering tasks seamlessly within your IDE or terminal
 </h1>
 
-<p align="center"><strong>goose</strong> <em>is a programming agent that runs on your machine.</em></p>
-
 <p align="center">
-  <a href="https://opensource.org/licenses/Apache-2.0"><img src="https://img.shields.io/badge/License-Apache_2.0-blue.svg"></a>
-  <a href="https://square.github.io/goose/"><img src="https://img.shields.io/badge/goose_documentation-blue"></a>
+  <img src="docs/assets/goose.png" width="400" height="400" alt="Goose Drawing"/>
+</p>
+<p align="center">
+  Generated by Goose from its <a href="https://github.com/square/goose-plugins/blob/main/src/goose_plugins/toolkits/artify.py">VincentVanCode toolkit</a>.
```

Comment:
can we add the canonical URL just in case we move this file from `HEAD`?

Reply:
added canonical url!